simple-service-webapp-rest/src/main/java/com/example/authentication/Token.java

```diff
@@ -0,0 +1,45 @@
+package com.example.authentication;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.util.Date;
+
+
+@JsonFormat(shape=JsonFormat.Shape.STRING, pattern="yyyy-MM-dd HH:mm a z")
+public class Token {
+
+    private String token;
+    private Date creationTime;
+
+    public Token() {
+    }
+
+    public Token(String token, Date creationTime) {
```

Comment:
Why not make a separate string field for the username? concatenating will be a pain to parse, the token string should just be the UUID generated string